templates/vue-demo-store/nuxt.config.ts

```diff
@@ -5,6 +5,9 @@ export default defineNuxtConfig({
     "@shopware-pwa/cms-base",
   ],
   runtimeConfig: {
+    public: {
+      broadcastChannelName: "shopware-composable-frotends",
+    },
```

Comment:
isn't that complicating config? It's an internal template mechanism; maybe it would be simpler to keep it with the mechanism itself?

Reply:
We can have many broadcast channels. I think this is the easiest way to change the name by the config